### DISCUSSION TITLE
Is there anybody out there?

### DISCUSSION THREAD

**Discussion by peteeckel** [auroraresearchlab/netbox-dns#56] [General]
Hi, 

please excuse me if I'm sounding a bit impatient, but after putting some effort into this project it seems to me that there is nobody actually looking at it - no comments, no questions, no requests for improvement, no reviews - nothing.

I understand if this is a small project done in spare time and there are other, more important issues to take care of. But it would really be great if there were at least some feedback telling me whether to go on or not - if my contributions are pushing in a direction that isn't intended, just tell me so and I will handle it, no hard thoughts.

I would like to use this tool for a project of mine, and if the development should be going nowhere that would mean that I have to fork to get the functionality I need finished, which would probably be an unnessesary duplication of work.

Best regards, 

  Peter.

**Comment by hbasria** [comment]
Hello Peter,

First of all, we apologize for the late feedback. Thank you very much for your development and support of the project.

This project is not a free time project, we want to support and develop for a long time. However, we could not make quick returns for intensity of our projects. We apologize again for this.

We started this project with the idea of how to make the DNS feature that is not available in nexbox  to record our company's domain inventory, we want to develop this according to the needs of the community without going beyond the scope of DNS.

We wish you continued support

best regards

**Comment by peteeckel** [comment]
HI Basri, 

no need to apologise, I've not taken any offense at all. 

I'm glad that we seem to be on the same page with our goals for this project. Apart from the customer project I will probably use it for my own DNS setup as well (and hence will be starting to look into the views issue as one of the next things as soon as I find the time), and so our planned usages are probably not far apart (if at all).

I will most certainly do what I can, and as soon as I can - it's going to be an extremely useful project and deserves the attention. Thank you for starting it!

Best regards, 

  Peter.
